<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IptcGroupBox</class>
 <widget class="QGroupBox" name="IptcGroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>160</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Check IPTC metadata</string>
  </property>
  <property name="checkable">
   <bool>true</bool>
  </property>
  <property name="checked">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="2">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>City</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="HistoryComboBox" name="countryNameComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Accepts regular expression</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Object name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>Country</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>Edit status</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="HistoryComboBox" name="cityComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Accepts regular expression</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Keywords</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="HistoryComboBox" name="objectNameComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Accepts regular expression</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="HistoryComboBox" name="editStatusComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Accepts regular expression</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="HistoryComboBox" name="descriptionComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Accepts regular expression</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="HistoryComboBox" name="keywordsComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Accepts regular expression</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HistoryComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/historycombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>objectNameComboBox</tabstop>
  <tabstop>keywordsComboBox</tabstop>
  <tabstop>descriptionComboBox</tabstop>
  <tabstop>countryNameComboBox</tabstop>
  <tabstop>cityComboBox</tabstop>
  <tabstop>editStatusComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
